src/drivers/indexeddb.js

```diff
@@ -1,766 +1,722 @@
+import createBlob from '../utils/createBlob';
+import idb from '../utils/idb';
+import Promise from '../utils/promise';
+import executeCallback from '../utils/executeCallback';
```

Comment:
Notice all the ES6 modules; this is gonna be a theme of my PR. Where possible I moved stuff to separate functions and then just `import`ed them.